src/pages/SchedulesPage.vue

```diff
@@ -143,32 +56,41 @@ export default {
         {
           id: 4,
           label: 'Active',
-          prop: 'active',
+          prop: 'activeLabel',
           isSortable: true,
           width: '135',
         },
       ],
     };
   },
   computed: {
+    ...mapGetters({ getSchedules: 'getSchedules' }),
     ...mapGetters({ getFormAction: 'getFormAction' }),
     modalTitle() {
       return this.getFormAction === 'create'
         ? 'Create Schedule'
         : 'Update Schedule';
     },
   },
+  mounted() {
+    this.fetchSchedules({ pageNumber: 1 });
```

Comment:
Use vuex bound value here
```suggestion
    this.fetchSchedules({ pageNumber: pagination.page });
```